### ISSUE TITLE
Show output in Console

### ISSUE THREAD

**Issue by laukstein** [SublimeLinter/SublimeLinter-stylelint#4]
Show output in Console similar like ESLint
![](https://raw.githubusercontent.com/stylelint/stylelint/master/example.png)

**Comment by braver** [comment]
Have you tried the SublimeLinter: Show All Errors command?

**Comment by laukstein** [comment]
@braver I don't think my Sublime Text with latest updates shows such an option, see

![grabilla q12712](https://user-images.githubusercontent.com/183966/39984176-fd288dee-5762-11e8-9090-5a72c4afbfef.png)

maybe is related to recent SublimeLinter or Sublime Text updates.

**Comment by braver** [comment]
Commands are in the command palette: 

<img width="541" alt="screen shot 2018-05-14 at 09 47 39" src="https://user-images.githubusercontent.com/2543659/39984543-dbb207a0-575b-11e8-8bd1-762ad10ea234.png">

https://www.sublimetext.com

**Comment by laukstein** [comment]
@braver it worked, thanks! `Ctrl+Shift+P` > "SublimeLinter: Show All Errors".
Can you please add stylelint related validation in right click menu, like you see "ESLint"? You could add "stylelint".

**Comment by braver** [comment]
> Can you please add stylelint related validation in right click menu

SublimeLinter only has generic commands, for any linter, so we cannot offer anything specific for stylelint (that's simply not how it works).

Also, the context menu only shows contextually relevant commands, like "Lint this view". We don't want to add commands to it because it will cause clutter for many users. Usually using the command palette is faster too.

That said, you can modify the context menu yourself: http://docs.sublimetext.info/en/latest/customization/menus.html

Basically, you add a `Context.sublime-menu` file to the `Packages/User/` directory, with the following contents:

```json
[
    {
        "id": "sublimelinter_panel",
        "caption": "Show All Errors",
        "command": "sublime_linter_panel_toggle"
    }
]
```
